vhdbuilder/packer/test/windows-vhd-content-test.ps1

```diff
@@ -213,9 +213,24 @@ function Test-DefenderSignature {
     }
 }
 
+function Test-AzureExtensions {
+    $expectedExtensions = @(
+        "Microsoft.CPlat.Core.RunCommandWindows"
```

Comment:
I think whether we can remove this extension in the last step.

Reply:
According to my investigation, it will be removed automatically after the pipeline.

Reply:
Additional investigation: this Windows test is called by ```az vm run-command invoke --command-id RunPowerShellScript```, and this extension is installed by it. So it's expected in this context, and it is not existing in windows node.

Reply:
Added notes for it.